app/lib/xos.py

```diff
@@ -5,13 +5,22 @@
 XOS_AUTH_TOKEN = os.environ['XOS_AUTH_TOKEN']
 XOS_API_ENDPOINT = os.environ['XOS_API_ENDPOINT']
 
-def update_xos_with_stub_video(video_data):
+def get_or_create_xos_stub_video(video_data):
```

Comment:
The logic of this is growing, could we add a docstring for this and the next function?